.circleci/config.yml

```diff
@@ -13,6 +13,8 @@ jobs:
       - restore_cache:
           key: v2-dependency-cache-{{ checksum "st2/requirements.txt" }}
       - run: sudo apt install python-dev
+      - run: sudo apt install libldap2-dev 
```

Comment:
This is great, could we also update the README.md to mention these extra dependencies so that people manually building docs will also know about the extra dependencies, e.g where it currently shows:
```
For Debian/Ubuntu: sudo apt-get install libpython-dev libssl-dev

For RHEL/CentOS: sudo yum install python-devel openssl-devel gcc
```

Reply:
Updated README.ms